src/qt/overviewpage.cpp

```diff
@@ -141,6 +141,18 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
     showOutOfSyncWarning(true);
     connect(ui->labelWalletStatus, &QPushButton::clicked, this, &OverviewPage::handleOutOfSyncWarningClicks);
     connect(ui->labelTransactionsStatus, &QPushButton::clicked, this, &OverviewPage::handleOutOfSyncWarningClicks);
+
+    QFont f = GUIUtil::fixedPitchFont();
+    f.setWeight(QFont::Bold);
+    f.setPointSize(10);
```

Comment:
Let's not override this for no reason?

Reply:
The only reason is to make things explicitly rather than implicitly. Does `setPointSize` call has any downsides?

Reply:
Yes, the downside is that it is explicit and overrides the user's preference (which may not be 10pt).

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/87#issuecomment-716163108).

Now I'm able to change the font size with the `qt5ct` utility on Linux Mint 20 :)